docs/modules/ROOT/pages/production-checklist.adoc

```diff
@@ -84,33 +81,23 @@ For Java 8;
 
+The number of internal partitions a Hazelcast member uses can be xref:ROOT:capacity-planning.adoc#partition-count[configured], but must be uniform across all members in the cluster.
 An optimal partition count and size establish a balance between the
-number partitions on each member and the data amount on each partition.
+number of partitions on each member and the data amount on each partition.
 You can consider the following when deciding on a partition count.
 
-* The partition count should be a prime number. This helps minimizing the collision of keys across
+* The partition count should be a prime number. This helps to minimize the collision of keys across
 partitions, ensuring more consistent lookup times.
-* A too low partition count is constraining; it should
+* A partition count which is too low constrains the cluster. The count should
 be large enough for a balanced data or task distribution so that each member
 does not manage too few partitions.
-* A partition size not exceeding 50MB typically ensures a good performance. For larger
-clusters, you can use the 50MB-100MB range; keep in mind that cluster performance may be affected
-adversely since larger heaps with more overhead space are required in this case.
+* A partition size of 50MB or less typically ensures good performance. Larger clusters may be able to use up to 100MB partition sizes, but will likely also require larger JVM heap sizes to accomodate the increase in data flow.
 
 If you are a Hazelcast Enterprise customer using the High-Density Data Store with large data sizes,
 we recommend a large increase in partition count, starting with 5009 or higher.
 
-The partition count cannot be changed after a cluster is created, so if you have
-a larger cluster, be sure to test and set an optimum partition count prior to deployment.
-If you need to change the partition count after a cluster is running, you will need to
-schedule a maintenance window to update the partition count and restart the cluster.
-
-== Large Cluster Configuration Recommendations
-
-You may need to increase the partition count relative to the number of Hazelcast members in the cluster and
-the data amount on each partition.
\ No newline at end of file
+The partition count cannot be eaily changed after a cluster is created, so if you have a large cluster be sure to test and set an optimum partition count prior to deployment. If you need to change th partition count after a cluster is already running, you will need to schedule a maintenance window to entirely bring the cluster down. If your cluster uses xref:storage:persistence.adoc[Persistence] or xref:cp-subsystem:persistence.adoc[CP Persistence] features, those persistent files will need to be removed after the cluster is shut down, as they contain references to the previous partition count. Once all member configurations are updated, and any persistent data structure files are removed, the cluster can be safely restarted.
```

Comment:
... cannot be easily changed ...

... If your cluster uses the xref:storage:persistence.adoc[Persistence] or xref:cp-subsystem:persistence.adoc[CP Persistence] feature, those ....

Reply:
Agreed, those should be fixed now.